CONTRIBUTING.md

```diff
@@ -1,41 +1,43 @@
 # How To Contribute
-Contributing to Gaia Dimension has various ways to contribute to the development. From a standpoint, contributions can include Coding, Assets and Testing. Please note that any PRs made can and will be rejected if they do not comply with the following guidelines. PRs will be closed if:
+There are various ways to contribute to the development of the Gaia Dimension mod. Some common types of contributions can include donating code, making textures, translating, and testing. Please note that any Pull Requests (PRs) made can and will be rejected if they do not comply with the following guidelines.
 
-* Does not comply with the listed guielines
-* No action is done to resolve conflicts or build failures within 2 weeks
-
-## Coding
-Though it is not easy from a surface level, it's worth more to contribute via coding than any other place. However, there are a few things to note when submitting a PR for coding.
+## Code
+Though it is not easy, donating code is the best way to contribute. However, there are a few things to note when submitting a PR with code.
 
 ### Make sure braces are on *the same line*
 As much as some people are inclined to, and for consistency's sake, make sure your code is formatted to use same-line braces. This is to reduce on file size and better vertical readability.
 
```

Comment:
There needs to be an additional paragraph for indentation type, specifically needs to be 4 spaces.

"Spacing must be 4 spaces: It seems trivial to bring this up, but please stick to 4 spaces per indentation as this is the project default. It helps when reviewing changes made to the source, as changes to indentations makes it appear like the entire file has been replaced."